docs/bigquery-usage.rst

```diff
@@ -281,7 +281,7 @@ Run a query which can be expected to complete within bounded time:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> query = """\
-   SELECT count(*) AS age_count FROM dataset_name.person_ages
+   SELECT count(*) AS age_count FROM [project-name:dataset_name.person_ages]
```

Comment:
Given that we don't otherwise document any other query syntax ourselves, I would just drop the project ID.